akka-http/src/main/mima-filters/10.1.8.backwards.excludes

```diff
@@ -9,4 +9,29 @@ ProblemFilters.exclude[IncompatibleResultTypeProblem]("akka.http.scaladsl.server
 ProblemFilters.exclude[ReversedMissingMethodProblem]("akka.http.scaladsl.server.directives.FormFieldDirectives#FieldMagnet.apply")
 ProblemFilters.exclude[IncompatibleResultTypeProblem]("akka.http.scaladsl.server.directives.FormFieldDirectives#FieldDef.apply")
 ProblemFilters.exclude[ReversedMissingMethodProblem]("akka.http.scaladsl.server.directives.FormFieldDirectives#FieldDef.apply")
-ProblemFilters.exclude[ReversedMissingMethodProblem]("akka.http.scaladsl.server.directives.FormFieldDirectives#FieldMagnet.convert")
\ No newline at end of file
+ProblemFilters.exclude[ReversedMissingMethodProblem]("akka.http.scaladsl.server.directives.FormFieldDirectives#FieldMagnet.convert")
+
+ProblemFilters.exclude[IncompatibleResultTypeProblem]("akka.http.scaladsl.server.directives.FormFieldDirectives.formFields")
+ProblemFilters.exclude[IncompatibleResultTypeProblem]("akka.http.scaladsl.server.directives.FormFieldDirectives.formField")
+ProblemFilters.exclude[IncompatibleResultTypeProblem]("akka.http.scaladsl.server.Directives.formFields")
+ProblemFilters.exclude[IncompatibleResultTypeProblem]("akka.http.scaladsl.server.Directives.formField")
+ProblemFilters.exclude[IncompatibleResultTypeProblem]("akka.http.scaladsl.server.HttpApp.formFields")
+ProblemFilters.exclude[IncompatibleResultTypeProblem]("akka.http.scaladsl.server.HttpApp.formField")
+
+ProblemFilters.exclude[IncompatibleSignatureProblem]("akka.http.scaladsl.server.directives.FormFieldDirectives#FieldDef.fieldDef")
+ProblemFilters.exclude[IncompatibleSignatureProblem]("akka.http.scaladsl.server.directives.FormFieldDirectives#FieldDef.forTuple")
```

Comment:
I'm not so sure the exclusions above, related to #2524, are safe. These only only apply to Scala 2.13.0, though.

Reply:
I added a module to test this particular case in https://github.com/akka/akka-http/pull/2597. AFAICS usages of `formField` are binary compatible. If someone would be dealing with `FieldDefs` manually that would fail (because the signature of its apply method changed).